src/test/java/org/fundacionjala/coding/ana/EanTest.java

```diff
@@ -0,0 +1,54 @@
+package org.fundacionjala.coding.ana;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * goods have an International Article Number.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class EanTest {
+    private Ean ean;
+
+    /**
+     * pruebas antes.
+     *
+     * @throws Exception antes.
+     */
+    @Before
+    public void setUp() {
+        ean = new Ean();
+    }
+
+    /**
+     * checking is true.
+     */
+    @Test
+    public void testEanArticleNumberTrue() {
+        boolean result = ean.articleNumber("4003301018398");
+        assertEquals(true, result);
+        System.out.print(result);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)